app/api.py

```diff
@@ -63,6 +63,42 @@ async def computer_assignment_rating_visualizer():
     return computer_assignment_visualizer(API.db)
 
 
+@API.post("/update/mentors")
+async def update(query: Dict, update_data: Dict):
+    """Updates Mentor documents that statisfy the query with update_data.
+
+    Queries from Mentor Collection with filters given (query).
+    Validate changes in update_data using MentorUpdate class (Pydantic schema)
+    and updates the corresponding fields, by overwriting or adding data.
+
+    Args:
+        query (dict): Key value pairs to filter for
+        update_data (dict): Key value pairs to update
+    Returns:
+        Updated fields or schema discrepancy error as dictionary
+    """
+    MentorUpdate(**update_data)
+    return {"result": API.db.update("Mentors", query, update_data)}
+
+
+@API.post("/update/mentees")
+async def update(query: Dict, update_data: Dict):
```

Comment:
same thing here, change the function name to something else and you should be good to go 👍